### DISCUSSION TITLE
Which terminals support this database?

### DISCUSSION THREAD

**Discussion by pipipapi** [GreptimeTeam/greptimedb#1889] [Q&A] [answered]
i think this one don't support it yet. so anyone else?

**Comment by pipipapi** [comment]
![image](https://github.com/GreptimeTeam/greptimedb/assets/29531009/20988d6a-2e18-4606-918c-31b6a8ccf4ec)
Which terminals support this database?

**Comment by pipipapi** [comment]
i think this one don't support it yet. so anyone else? 
..Jesus ，github lost fisrt image uploaded just a mount ago。

**Reply by MichaelScofield** [reply, marked as answer]
We suggest using `mysql` cli or `psql` cli to connect to GreptimeDB. Other GUI tools might just work, too. For example, from your screenshot, you could choose "MySQL", "SQLite" or "PostgreSQL". However, some GUI tools will emit custom SQLs upon connection which we might not support.

PS: You can find our supported client protocols [here](https://docs.greptime.com/user-guide/clients/overview).
